### PR TITLE
Move 70-persistent-net.rules instead of remove it

### PR DESCRIPTION
**Pull request by imobach**

### CODE REVIEW

File: share/veewee/postinstall.sh [hunk 1]
```diff
@@ -27,9 +27,10 @@ echo -e "\nspeed-up remote logins ..."
 echo -e "\n# added by veewee/postinstall.sh" >> /etc/ssh/sshd_config
 echo -e "UseDNS no\n" >> /etc/ssh/sshd_config
 
-# avoid mac address configured into system, this results in getting
-# eth1 instead of eth0 in virtualized environments sometimes
-rm -f /etc/udev/rules.d/70-persistent-net.rules
+# Avoid mac address configured into system, this results in getting
+# eth1 instead of eth0 in virtualized environments sometimes.
+# Anyway it won't be removed so we can check those rules.
+mv /etc/udev/rules.d/70-persistent-net.rules{,.old}
```
Comment (ancorgs) [inline share/veewee/postinstall.sh]: Maybe `.old` is a too common profile that could be used by something else. Let's use something like `.aytests`

Comment (imobach) [inline share/veewee/postinstall.sh]: Ok, it makes sense. Updated.

**Comment by ancorgs** [comment]
LGTM

**Comment by imobach** [comment]
Thanks!

**Pull request merged**